dclab/kde/base.py

```diff
@@ -120,6 +125,96 @@ def get_contour(self, xax="area_um", yax="deform", xacc=None, yacc=None,
             xscale=xscale, yscale=yscale
         )
 
+    def get_contour_lines(self, xax="area_um", yax="deform", xacc=None,
+                          yacc=None, kde_type="histogram", kde_kwargs=None,
+                          xscale="linear", yscale="linear", ret_levels=False,
+                          quantiles=None):
+        """Compute contour lines for a given kernel kensity estimate.
+
+        Parameters
+        ----------
+        xax: str
+            Identifier for X axis (e.g. "area_um", "aspect", "deform")
+        yax: str
+            Identifier for Y axis
+        xacc: float
+            Contour accuracy in x direction
+            if set to None, will use :func:`bin_width_doane_div5`
+        yacc: float
+            Contour accuracy in y direction
+            if set to None, will use :func:`bin_width_doane_div5`
+        kde_type: str
+            The KDE method to use
+        kde_kwargs: dict
+            Additional keyword arguments to the KDE method
+        xscale: str
+            If set to "log", take the logarithm of the x-values before
+            computing the KDE. This is useful when data are
+            displayed on a log-scale. Defaults to "linear".
+        yscale: str
+            See `xscale`
+        ret_levels: bool
+            If set to True, return the levels of the contours
+            (default: False)
+        quantiles: list or array of floats
+            Define the levels of the contours. The values must be between 0
```

Comment:
The first sentence should explicitly mention quantiles. "Define the contour levels based on quantiles of the KDE."